File: frontend/src/products/Product.js

const Product = ({props, onClick}) => {
    const { creationDate, description, id, imagePath, isActive, price, quantity, title } = props;
    return(
        <>
          <div className="product-container">
          <div className="product-image">
            <img src={imagePath}></img>
          </div>
          <div className="product-details">
            <div >{title}</div>
            <div >{description}</div>
            <div >{price} $</div>
            <div>{quantity} pcs</div>
          </div>
          <div className="product-button">
            <button onClick={onClick} className="primary-button">Edit</button>
          </div>
        </div>
        </>
    )
}

export default Product;

File: backend/Models/Product/Product.cs
using System;
using System.Collections.Generic;

namespace backend.Models.Product;

public partial class Product
{
    public int Id { get; set; }

    public string Title { get; set; } = null!;

    public string Description { get; set; } = null!;

    public decimal Price { get; set; }

    public int Quantity { get; set; }

    public string? ImagePath { get; set; }

    public DateTime CreationDate { get; set; }

    public bool IsActive { get; set; }
}
